build/build.go

```diff
@@ -105,38 +116,35 @@ func BundleSharedScripts() (string, error) {
 var evalScriptTemplate string
 
 // BuildEvalScriptFromFile gets a script from a file and builds an eval script with it.
-func BuildEvalScriptFromFile(serverPort string, uiMode cdp.UIMode, scriptPath, steamPath, authToken, pluginsDir string) (string, error) {
+func BuildEvalScriptFromFile(serverPort string, uiMode cdp.UIMode, scriptPath, steamPath, authToken, pluginsDir string, platform string) (string, error) {
 	scriptBytes, err := ioutil.ReadFile(scriptPath)
 	if err != nil {
 		return "", fmt.Errorf(`Failed to read injected script at "%s": %w`, scriptPath, err)
 	}
 
 	script := string(scriptBytes)
 
-	return BuildEvalScript(serverPort, uiMode, script, steamPath, authToken, pluginsDir)
+	return BuildEvalScript(serverPort, uiMode, script, steamPath, authToken, pluginsDir, platform)
 }
 
 // BuildEvalScript builds a script to be evaluated in the Steam target context.
-func BuildEvalScript(serverPort string, uiMode cdp.UIMode, script, steamPath, authToken, pluginsDir string) (string, error) {
+func BuildEvalScript(serverPort string, uiMode cdp.UIMode, script, steamPath, authToken, pluginsDir string, platform string) (string, error) {
 	evalTmpl := template.Must(template.New("eval").Parse(evalScriptTemplate))
 	var evalScript bytes.Buffer
-	if err := evalTmpl.Execute(&evalScript, struct {
-		Version        string
-		InjectedScript string
-		ServerPort     string
-		UIMode         cdp.UIMode
-		SteamDir       string
-		AuthToken      string
-		PluginsDir     string
-	}{
+
+	// Create globals struct for script execution
+	globals := csGlobals{
 		Version:        VERSION,
 		InjectedScript: script,
 		ServerPort:     serverPort,
 		UIMode:         uiMode,
 		SteamDir:       steamPath,
 		AuthToken:      authToken,
 		PluginsDir:     pluginsDir,
-	}); err != nil {
+		Platform:       platform,
```

Comment:
I think we can directly set `Platform` here so we don't have to pass it in from every build call

```suggestion
		Platform:       runtime.GOOS,
```